### PR TITLE
(Minor) Exit on error and report argument error details

### PR DESCRIPTION
**Pull request by achalddave**
* The statement 'exit' has no effect in Python scripts - it is treated as a variable reference. Use sys.exit() instead.
* Strip trailing whitespaces.

### CODE REVIEW

**Comment by seanbell** [comment]
LGTM

**Comment by longjon** [comment]
Thanks, these changes are correct (how did this ever end up in merged code?).

However, to keep the history clean, could you split this into two commits, one for the code changes, and one for the whitespace changes? As a general rule, it's best to avoid commits which both change code and change (unrelated) whitespace.

File: tools/extra/plot_training_log.py.example [hunk 1]
```diff
@@ -68,9 +68,9 @@ def get_field_descriptions(chart_type):
         get_chart_type_description_separator())
     y_axis_field = description[0]
     x_axis_field = description[1]
-    return x_axis_field, y_axis_field    
+    return x_axis_field, y_axis_field
 
-def get_field_indecies(x_axis_field, y_axis_field):    
+def get_field_indecies(x_axis_field, y_axis_field):
```
Comment (shelhamer) [inline tools/extra/plot_training_log.py.example]: Spelling: `indecies` -> `indices`

**Comment by achalddave** [comment]
Sounds good. I tend to make too many commits in my own repos so perhaps I
overcorrected here :)

Also, there are a number of small typos/formatting issues in this script.
Should I fix them in this PR while I'm at it?

**Comment by achalddave** [comment]
Okay, I've fixed minor unrelated things in this PR (trailing spaces, typos, etc.). I haven't done any other major things (significantly re-formatting code / using arpgarse instead of sys.argv).

**Comment by longjon** [comment]
Great, this history looks pretty clean now. This PR is ready for merge, so if you've got more fixes you'd like to send, we'd be happy to see additional PRs. Thanks!

**Pull request merged**